### DISCUSSION TITLE
longer text display in fields in result view

### DISCUSSION THREAD

**Discussion by littlegreenbean33** [tconbeer/harlequin#400] [Q&A, Troubleshooting, Support] [answered]
Hi,

I have a table with some long text fields that I would like to be able to see in full ( after pressing F10). Right now, long text fields (i.e. json) are truncated after some length.

Where is this parameter set so I can modify it in my configuration file ? 

Thanks

JC

**Comment by tconbeer** [comment, marked as answer]
This isn't configurable today; you can hover with your mouse to see the full value, or you can copy the value and paste into a text editor.

**Reply by tconbeer** [reply]
@littlegreenbean33 does this solution work for you, or would you still like to see a change made to Harlequin here?

**Reply by littlegreenbean33** [reply]
That is fine. Many Thanks.
